### ISSUE TITLE
Broken logos

### ISSUE THREAD

**Issue by WillemElbers** [clarin-eric/discovery-service-frontend#24]
Logos which cannot be loaded currently show "Logo" as the alt text.

**Comment by WillemElbers** [comment]
Broken logos are now hidden. Fixed in 79e7015
